code/modules/power/supermatter/supermatter.dm

```diff
@@ -476,6 +476,8 @@
 /obj/machinery/power/supermatter_shard/attack_paw(mob/user)
 	return attack_hand(user)
 
+/obj/machinery/power/supermatter_shard/attack_animal(mob/living/simple_animal/user)
+	return attack_hand(user)
```

Comment:
@RemieRichards I'm more of a stickler for the semantics of it.

The object was not attack_hand'd it was attack_animal'd.

In turn, both of those actions cause the user do die horribly, therefore both procs should call a proc called die_horribly() or whatever.